CMakeLists.txt

```diff
@@ -1114,8 +1114,11 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/zeek-version.h.in
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/zeek-version.h DESTINATION include/zeek)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/zeek-config.h.in
                ${CMAKE_CURRENT_BINARY_DIR}/zeek-config.h)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/zeek-config-paths.h.in
+               ${CMAKE_CURRENT_BINARY_DIR}/zeek-config-paths.h)
 include_directories(BEFORE ${CMAKE_CURRENT_BINARY_DIR})
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/zeek-config.h DESTINATION include/zeek)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/zeek-config.h
+              ${CMAKE_CURRENT_BINARY_DIR}/zeek-config-paths.h DESTINATION include/zeek)
```

Comment:
This works, but lumping two headers with separate `configure_file` into one `install` makes it slightly harder to follow what is going on.

Could you clean this up a little so we have stuff belonging together in groups, maybe even separated by some extra whitespace, e.g.,

```cmake
configure_file(...)
install(...)

configure_file(...)
install(...)

...
```

This should make it pretty obvious that we are installing all of them.